src/utils/daos_hdlr.c

```diff
@@ -3607,30 +3316,27 @@ cont_clone_hdlr(struct cmd_args_s *ap)
 	src_str_len = strlen(ap->src);
 	D_STRNDUP(src_str, ap->src, src_str_len);
 	if (src_str == NULL) {
-		fprintf(ap->errstream,
-			"Unable to allocate memory for source path.");
-		D_GOTO(out, rc = -DER_NOMEM);
+		rc = -DER_NOMEM;
+		DH_PERROR_DER(ap, rc, "Unable to allocate memory for source path");
+		D_GOTO(out, rc);
 	}
-	rc = dm_parse_path(&src_cp_type, src_str, src_str_len,
-			   &ca.src_p_uuid, &ca.src_c_uuid);
+	rc = dm_parse_path(&src_cp_type, src_str, src_str_len, &ca.src_p_uuid, &ca.src_c_uuid);
 	if (rc != 0) {
-		fprintf(ap->errstream, "failed to parse source path: %d\n", rc);
-		D_GOTO(out, rc = daos_errno2der(rc));
+		DH_PERROR_DER(ap, rc, "Failed to parse source path");
+		D_GOTO(out, rc);
 	}
 
 	dst_str_len = strlen(ap->dst);
 	D_STRNDUP(dst_str, ap->dst, dst_str_len);
 	if (dst_str == NULL) {
-		fprintf(ap->errstream,
-			"Unable to allocate memory for dest path.");
+		rc = -DER_NOMEM;
+		DH_PERROR_DER(ap, rc, "Unable to allocate memory for destination path");
 		D_GOTO(out, rc = -DER_NOMEM);
```

Comment:
You're setting rc twice here.  Minor ;)
```suggestion
		D_GOTO(out, rc);
```